docs/deploy_features.md

```diff
@@ -19,6 +20,23 @@ We generally find that most developers are able to get high quality answers usin
 * In `infra/main.bicep`, change `chatGptModelName` to 'gpt-4' instead of 'gpt-35-turbo'.
 * You may also need to adjust the capacity above that line depending on how much TPM your account is allowed.
 
+## Using ada-3 embedding models
```

Comment:
Can you please add a note that this isn't supported with int vectorization?

Reply:
I added that note under the int vect heading, but I could also note here.